docs/encyclopedia/application-message-passing.mdx

```diff
@@ -53,7 +57,7 @@ Use Signals instead of Updates when:
 - The Workflow's clients want to quickly move on after sending an asynchronous message.
 - The clients care more about the latency of sending the message than they do about the latency of the end-to-end operation.
 - The clients don't want to rely on the Worker being available.
-- Clients need to [Signal-With-Start](#signal-with-start) atomically (since there is currently no Update-with-Start operation.)
+- Clients need to [Signal-With-Start](#signal-with-start) atomically (since Update-with-Start is not atomic).
```

Comment:
I vote to cut this.  When I wrote it initially, I really meant "atomic" imprecisely as "one API call."
The number of users who specifically care about atomicity is probably small enough that this gotcha can be relegated to the more detailed look at UwS.